crates/pop-contracts/src/utils/metadata.rs

```diff
@@ -153,26 +300,26 @@ pub fn process_function_args<P>(
 where
 	P: AsRef<Path>,
 {
-	let parsed_args = args.into_iter().map(|arg| arg.replace(",", "")).collect::<Vec<_>>();
```

Comment:
Note that this breaks entering complex types, where fields are separated by commas.